website/docs/functions/validate-text.md

```diff
@@ -13,18 +13,21 @@ Option | Default | Possible values
 `medium` | `all` | `all`, `braille`, `embossed`, `handheld`, `print`, `projection`, `screen`, `speech`, `tty`, `tv`
 `warningLevel` | `0` | `0`, `1`, `2`, `3`
 `timeout` | `10000` | `integer`
+`profile` | `css3` | `none`, `css1`, `css2`, `css21`, `css3`, `css3svg`, `svg`, `svgbasic`, `svgtiny`, `mobile`, `atsc-tv`, `tv`
 
 Option | Explanation
 :--- | :---
 `medium` | The equivalent of the `@media` rule, applied to all of the CSS
 `warningLevel` | `0` means don’t return any warnings; `1`, `2`, `3` will return warnings (if any), with higher numbers corresponding to more warnings
 `timeout` | The time in milliseconds after which the request to the W3C API will be terminated and an error will be thrown
+`profile` | Specifies the validation profile to use for CSS validation
```

Comment:
I'd like to make this a bit more helpful for someone who might not be familiar with what a CSS profile is, and why they might want to customize this option. Plus, it'd be helpful to link to W3C's manual so users can read more about the option:

```suggestion
`profile` | The CSS profile to use for validation, which specifies the available CSS features; [learn more](https://jigsaw.w3.org/css-validator/manual.html#paramprofile)
```